common/math/geometry/src/spline/catmull_rom_subspline.cpp

```diff
@@ -23,6 +23,15 @@ namespace geometry
 {
 double CatmullRomSubspline::getLength() const { return end_s_ - start_s_; }
 
+std::pair<double, double> CatmullRomSubspline::getAltitudeRange() const
+{
+  if (spline_) {
+    return spline_->getAltitudeRange();
```

Comment:
This returns the altitude range of the entire spline, not the subspline segment [ `start_s_,` `end_s_` ]. This seems inconsistent with other methods like `getLength()` and `getCollisionPointIn2D()` which correctly respect the subspline bounds.